### PR TITLE
fix(docs): Add Slices pattern example

### PR DESCRIPTION
**Pull request by jmantonellini**
By ading this example we can combine the real Slices pattern implementation (instead of multiple stores) with the resetAll method.
Hope this helps! 😄

## Related Issues

Fixes #.

## Check List

- [ ] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Comment by dai-shi** [comment]
@chrisk-7777 can you review this? (slice pattern might not still be 100% stable.)

**Comment by dai-shi** [comment]
@sewera whenever you have time.

File: docs/guides/how-to-reset-state.md [hunk 1]
```diff
@@ -74,6 +74,72 @@ export const resetAllSlices = () => {
 }
 ```
 
+Resetting bound store using Slices pettern
```
Comment (sewera) [inline docs/guides/how-to-reset-state.md:77]: pettern => pattern

**Review by sewera — COMMENTED** [review]
Not a real review, but I can drop in with some easy, actionable task while on my phone:

- Please align the code style with the one in the Zustand code base.
- Please remove the comments. They are redundant, as the variable names provide sufficient documentation. If you feel, however, that the code blocks could be named, consider extracting functions for them. I don't feel that's justified here, though.

**Comment by sewera** [comment]
Hi @jmantonellini,

what's the status of this PR?

Do we continue with it? I didn't put any other comments,
because I didn't get any feedback on the existing ones.

Anyway, please let me know if this PR is still relevant :)

**Comment by jmantonellini** [comment]
Hi @sewera !

Thank you for reviewing this.

I commited the changes that you requested.

File: docs/guides/how-to-reset-state.md [hunk 2]
```diff
@@ -74,6 +74,70 @@ export const resetAllSlices = () => {
 }
 ```
 
+Resetting bound store using Slices pattern
+
+```ts
```
Comment (sewera) [inline docs/guides/how-to-reset-state.md:79]: Please align the code style to the rest of code snippets (running Prettier may do this automatically for you, see the task in PR description).

File: docs/guides/how-to-reset-state.md [hunk 3]
```diff
@@ -74,6 +74,70 @@ export const resetAllSlices = () => {
...
+const useBoundStore = create<BearSlice & FishSlice>()((...a) => ({
+  ...createBearSlice(...a),
+  ...createFishSlice(...a),
+}));
+
+export const resetAllSlices = () => {
+  for (const resetter of resetters) {
+    resetter();
+  }
+};
```
Comment (sewera) [inline docs/guides/how-to-reset-state.md:136]:
```suggestion
export const resetAllSlices = () => resetters.forEach(resetter => resetter())
```

**Review by sewera — CHANGES_REQUESTED** [review]
Apart from the comments in code, this looks about right.

File: docs/guides/how-to-reset-state.md (hunk 3 above)
Comment (jmantonellini) [reply docs/guides/how-to-reset-state.md]: Thank you for your review!

File: docs/guides/how-to-reset-state.md (hunk 2 above)
Comment (jmantonellini) [reply docs/guides/how-to-reset-state.md:79]: Done! 😄 Thank you

File: docs/guides/how-to-reset-state.md [hunk 4]
```diff
@@ -74,6 +74,70 @@ export const resetAllSlices = () => {
...
 ```
 
+Resetting bound store using Slices pattern
+
+```ts
+import create, { StateCreator } from "zustand";
+
+const resetters: (() => void)[] = [];
+
+const initialStateBear = { bears: 0 };
```
[the remark this answers is not in the document]
Comment (jmantonellini) [reply docs/guides/how-to-reset-state.md]: Done ✅

**Comment by jmantonellini** [comment]
Last changes already pushed ✅

Should be good to go now @sewera

**Review by sewera — APPROVED** [review]

**Pull request merged**